docs/DesReqGui/source/core_kpi.rst

```diff
@@ -51,6 +51,23 @@ Energy use in a given period of time
    any type; finally, :math:`P_i` is the instantaneous power used by the
    energy vector :math:`i`; :math:`A` is the total floor area of the building.
 
+Peak electricity demand in a given period of time
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+   This KPI is the measure of the site HVAC peak electricity use in :math:`kW/m^2`
+   when accounting for the sum of all electrical equiipment present in the test
+   case HVAC system.
+   The mathematical formulation for this KPI is the following:
+
+   .. math:: P_e(t_0, t_f) = \frac{\max_{15}[\sum_{i\in \epsilon} P_i(t)]}{A}
+
+   Where :math:`P_e(t_0, t_f)` is the peak electricity demand from the
+   initial time :math:`t_0` up to the final time :math:`t_f`; :math:`\epsilon` denotes
+   the set of equipment in the system with an associated energy use of
+   electricity; :math:`P_i` is the instantaneous power used by the
```

Comment:
It is a bit strange that we talk here about a generic energy vector  `i` when in this case there is only one energy vector which is electricity. I would reformulate using just `P`, the instantaneous electricity use.

Reply:
How about my suggestion in https://github.com/ibpsa/project1-boptest/pull/394/commits/b03ed26fbc7df72babe727260a0f83cce6afc7ea to just make the set of all equipment ``I``?  And I'd suggest just redefining ``I`` locally for Gas and District peaks.

Reply:
Looking good. Thanks for taking care of it!